test/com/facebook/buck/jvm/kotlin/testdata/kotlin_library_description/com/example/mixed/Example.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.good
```

Comment:
They would also need to be compiled and run with a version of kotlinc 1.1.3 or higher in order to pass.